src/cvl/schema/sonic-ifa.yang

```diff
@@ -0,0 +1,126 @@
+module sonic-ifa {
+    namespace "http://github.com/Azure/sonic-ifa";
+    prefix ifa;
+
+    import ietf-inet-types {
```

Comment:
Please remove the yang from cvl/schema, it needs to be placed only in models/yang/sonic

Reply:
Done